carbon-design-system/carbon-platform
# chore: save prototyping data

Commits prototyping data to this repository so we can host the prototype on Vercel without Vercel needing to make GitHub API requests to either GitHub public or enterprise.

This results in 498 JSON files (9.2MB) being committed to the repository.

#### Changelog

**New**

- `/data/libraries.js` specifies Carbon charts and components libraries to include
- `/config/constants.js` uses the platform run time, but defaults to `.carbon-oss`, so Vercel shouldn't need any env vars set

**Changed**

- By defaults uses the new `.carbon-oss` file system cache directory. To include the full data set, set `CARBON_RUN_MODE=PRODUCTION`

**Removed**

- N/A

#### Testing / reviewing

1. Run clean and dev. You should only see charts and components libraries.
2. Set `CARBON_RUN_MODE=PRODUCTION`, clean, dev, and you should see all libraries like before.
3. PRO MODE TEST turn off wifi / internet and do a build (it works, meaning it isn't reaching out to GitHub APIs anywhere)
